### PR TITLE
Added "outdated" banner to every docs page

### PR DESCRIPTION
**Pull request by RFMVasconcelos**
Adding "outdated banner" to every page on master branch to enforce reviews by owners - following request in #1984.

### CODE REVIEW

File: content/en/docs/about/style-guide.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
 
 +++
+
+{{% alert title="Out of date" color="warning" %}}
```
Comment (joeliedtke) [inline content/en/docs/about/style-guide.md:4]: Please remove this alert. This is affecting the preview deployment.

Comment (RFMVasconcelos) [reply content/en/docs/about/style-guide.md]: Done

**Comment by joeliedtke** [comment]
Preview:
https://deploy-preview-2031--competent-brattain-de2d6d.netlify.app/docs/

File: content/en/docs/_index.md [hunk 2]
```diff
@@ -2,3 +2,9 @@
 title = "Documentation"
 description = "All of Kubeflow documentation"
 +++
+
+{{% alert title="Out of date" color="warning" %}}
```
Comment (joeliedtke) [inline content/en/docs/_index.md:6]: _index.md files mainly contain content from other pages. Would it be possible to exclude the _index.md pages from this PR?

File: content/en/docs/about/_index.md [hunk 3]
```diff
@@ -3,3 +3,9 @@ title = "About"
 description = "Information about Kubeflow software, community, docs, and events."
 weight = 10
 +++
+
+{{% alert title="Out of date" color="warning" %}}
```
Comment (joeliedtke) [inline content/en/docs/about/_index.md:7]: The About Kubeflow section doesn't appear to have version specific content. Would it be possible to exclude this section?

File: content/en/docs/_index.md (hunk 2 above)
Comment (RFMVasconcelos) [reply content/en/docs/_index.md:6]: Hi @joeliedtke, sure!

**Comment by jlewi** [comment]
@RFMVasconcelos it looks like this PR has been stuck for a while? Do you plan on fixing this and merging? It looks like the CLA is failing because the author on your commit history is not set; if you look at your first commit it looks like its not mapping to your GitHub account probably because the email isn't set.

Usually rebasing can fix that.  Instructions in this issue might help
https://github.com/kubeflow/website/issues/1094

However, given that people have started updating the docs do we still want to submit this as people might have already updated the docs at this point and might not be happy about that.

Is it possible to update your script to only add the banner to pages that haven't been updated say in the last 30 days?

**Comment by RFMVasconcelos** [comment]
Hi @jlewi, 

Sorry, I did not realize that it was breaking because of that. 

As for updating the script to checking for latest changes, let me jump on that. Would it be more efficient to submit a new PR with this change?

**Comment by RFMVasconcelos** [comment]
@jlewi I have the script to add the outdated banners only for not edited in last 30 days docs. Will commit that tomorrow.

**Comment by RFMVasconcelos** [comment]
> @RFMVasconcelos it looks like this PR has been stuck for a while? Do you plan on fixing this and merging? It looks like the CLA is failing because the author on your commit history is not set; if you look at your first commit it looks like its not mapping to your GitHub account probably because the email isn't set.
> 
> Usually rebasing can fix that. Instructions in this issue might help
> #1094
> 
> However, given that people have started updating the docs do we still want to submit this as people might have already updated the docs at this point and might not be happy about that.
> 
> Is it possible to update your script to only add the banner to pages that haven't been updated say in the last 30 days?

**Comment by RFMVasconcelos** [comment]
Closing this PR, to replace with #2066

**Pull request closed without merging**